msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.ObjCBinding.Common.props

```diff
@@ -17,11 +17,24 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 	<Import Project="$(MSBuildThisFileDirectory)$(MSBuildThisFileName).Before.props"
 			Condition="Exists('$(MSBuildThisFileDirectory)$(MSBuildThisFileName).Before.props')"/>
 
-	<!-- This is used to determine whether Xamarin.Mac.Common.targets needs to import 
+	<!-- This is used to determine whether Xamarin.Mac.ObjCBinding.Common.targets needs to import 
          Xamarin.Mac.ObjCBinding.Common.props itself, or whether it has been imported previously,
          e.g. by the project itself. -->
 	<PropertyGroup>
-		<_XamarinCommonPropsHasBeenImported>true</_XamarinCommonPropsHasBeenImported>
```

Comment:
We were getting "lucky" with having two files abusing same double include guard. Now that we pull in the other copy it doesn't work now (and wasn't safe before).

Reply:
good catch, then!